src/azure-cli/azure/cli/command_modules/iot/tests/hybrid_2020_09_01/recording_processors.py

```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.scenario_tests import RecordingProcessor
+from azure_devtools.scenario_tests.utilities import is_text_payload
+
+MOCK_KEY = 'mock_key'
+
+
+class KeyReplacer(RecordingProcessor):
```

Comment:
May I know which kind of key to replace here？storage account key? If yes, I think we already support in https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-testsdk/azure/cli/testsdk/base.py#L87